src/bp/core/services/middleware/event-engine.ts

```diff
@@ -103,10 +108,10 @@ export class EventEngine {
 
   /**
    * Outputs to the console the event I/O in real-time
-   * Usage: set `BP_DEBUG_IO` to `true`
+   * Usage: `DEBUG=bp:middleware:performance*`
    */
   private setupPerformanceHooks() {
-    if (!process.env.BP_DEBUG_IO) {
+    if (!debugPerformance.enabled) {
```

Comment:
i'm not totally sure about this one. I think it would be best kept in an environment variable. This is only called when the server is started, so it wouldn't be updated if you change it afterwards. 

Plus, it's very rare that we will use these. I think people will often want to use bp:* to debug, and performance hooks aren't useful there

Reply:
Make sense, I'll bring back the env var